build/buildall

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+# Install all the versions we support
+mvn -U -Dbuildver=301 clean install -Drat.skip=true -DskipTests
+mvn -U -Dbuildver=302 clean install -Drat.skip=true -DskipTests
+mvn -U -Dbuildver=303 clean install -Drat.skip=true -DskipTests
+mvn -U -Dbuildver=304 clean install -Drat.skip=true -DskipTests
+mvn -U -Dbuildver=311 clean install -Drat.skip=true -DskipTests
+mvn -U -Dbuildver=312 clean install -Drat.skip=true -DskipTests
+mvn -U -Dbuildver=313 clean install -Drat.skip=true -DskipTests
```

Comment:
are we missing 320 in here?

Reply:
left it out intentionally since audit stuff and tests not all passing.  I can add it in if we want.

note I left it out of nightly too mostly since I wasn't sure if tests work there.

Reply:
since  incompatibilities  in the non-patch releases is the main motivation for this whole epic, it's important that when we test  320 (even if just locally) we test it with a combined jar including at least one 3.0.x and a at least one 3.1.y

Reply:
added it